crates/circuit/src/dag_circuit.rs

```diff
@@ -1479,29 +1472,22 @@ impl DAGCircuit {
     ///     DAGCircuitError: a qreg is not a QuantumRegister, or is not in
     ///     the circuit.
     #[pyo3(signature = (*qregs))]
-    fn remove_qregs(&mut self, qregs: &Bound<PyTuple>) -> PyResult<()> {
+    fn remove_qregs(&mut self, qregs: Vec<QuantumRegister>) -> PyResult<()> {
```

Comment:
Same comment here.

Reply:
I split this out into a rust and python side function in: https://github.com/Qiskit/qiskit/pull/14313/commits/9a43895766979f8ee3667248ba0f754f76392cb8